docusaurus.config.js

```diff
@@ -39,7 +39,7 @@ const config = {
           // Please change this to your repo.
           // Remove this to remove the "edit this page" links.
           editUrl:
-            'https://github.com/BattlesnakeOfficial/docs',
+            'https://github.com/BattlesnakeOfficial/docs/tree/main',
```

Comment:
```suggestion
            'https://github.com/BattlesnakeOfficial/docs/blob/main',
```

`blob` is for links that point to a file, `tree` is for links that point to a directory.

Reply:
Interestingly, the official docusaurus docs use `edit/main`.
https://github.com/facebook/docusaurus/blob/main/website/docusaurus.config.js#L172

Which works for me, but not sure what the ux is for non-admins?

Reply:
I tested from my automation account and got: 
<img width="865" alt="image" src="https://user-images.githubusercontent.com/63889819/187013214-e1bc7bc1-204b-4839-ba13-c80c8617c892.png">

Reply:
Setting `editUrl` to:
https://github.com/BattlesnakeOfficial/docs/edit/main 

would be the best UX for non-admins, as this will ask them to fork the repository. like Naomi showed above.

Reply:
This should be fixed in the latest commit